doc/theoretical_description_regression.rst

```diff
@@ -288,4 +340,8 @@ References
 34th Conference on Neural Information Processing Systems (NeurIPS 2020).
 
 [3] Yaniv Romano, Evan Patterson, Emmanuel J. Candès.
-"Conformalized Quantile Regression." Advances in neural information processing systems 32 (2019).
\ No newline at end of file
+"Conformalized Quantile Regression." Advances in neural information processing systems 32 (2019).
+
+[7] Chen Xu, Yao Xie.
+"Conformal prediction for dynamic time-series"
+https://arxiv.org/abs/2010.09107
```

Comment:
Can you cite this paper as Chen Xu suggested (also in the README) ? 
Chen Xu and Yao Xie. "Conformal Prediction Interval for Dynamic Time-Series.” International Conference on Machine Learning (ICML) 2021 (long presentation).

Reply:
Done